Hardcode buildsystem to use /app. Obviously shouldn't be merged upstream.

---
diff --git a/DEFAULTS/Defaults.linux b/DEFAULTS/Defaults.linux
index bbc460df..74493bb5 100644
--- a/DEFAULTS/Defaults.linux
+++ b/DEFAULTS/Defaults.linux
@@ -41,9 +41,9 @@ LINUX_SRC_INCLUDE=      $(__LINUX_SRC_INCLUDE:$(_UNIQ)no%=%)
 
 DEFINCDIRS=	$(SRCROOT)/include
 DEFOSINCDIRS=
-LDPATH=		-L/opt/schily/lib
-#RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib -R$(OLIBSDIR)
-RUNPATH=	-R$(INS_BASE)/lib -R/opt/schily/lib
+LDPATH=		-L/app/lib
+#RUNPATH=	-R$(INS_BASE)/lib -R/app/lib -R$(OLIBSDIR)
+RUNPATH=	-R$(INS_BASE)/lib -R/app/lib
 
 ###########################################################################
 #
@@ -57,7 +57,7 @@ OSDEFS +=	-D_GNU_SOURCE
 # Installation config stuff
 #
 ###########################################################################
-INS_BASE=	/opt/schily
+INS_BASE=	/app
 INS_KBASE=	/
 INS_RBASE=	/
 #
